renderers/server-render.jsx

```diff
@@ -0,0 +1,172 @@
+import Rx from 'rxjs';
```

Comment:
yep yep. let me do one more quick pass at this. i think this is mergeable. then i'll jump over to mup-web.